plotify/plotify.py

```
@@ -60,6 +65,26 @@ def _get_column_cardinality(df, column_name):
 
     return df[column_name].nunique()
 
+def _check_valid_calculation(some_dict):
+    try:
```

Why the try/except? 
you can use `.get` so missing key still works

---

I want to raise an exception if the dict is not valid. I don;t know what would be a good behavior otherwise

---

👍